roles/aws/admin-route53/tasks/adm-record.yml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
In general, we should do remove first then add - i.e.: if an record needs to be updated. (it's also done this way in the `dns` role)

Reply:
done